### ISSUE TITLE
[a11y]: ProgressIndicator accessibility error on span cds--progress-line

### ISSUE THREAD

**Issue by David-Yuen** [carbon-design-system/carbon#17916]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.68.0

### React version

18.3.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

`ProgressIndicator`/`ProgressStep` component has a `span` element with class `cds--progress-line` is throwing an accessibility error:
```
Undersized target "span" does not have sufficient spacing of 12 CSS pixels from another target "span"
```

I'm not sure if this is bug with the scan tool or a way to resolve this.. doesn't look configurable through the carbon library props (i think).

Here is screenshot:
![Image](https://github.com/user-attachments/assets/a000d29a-fde8-4cc6-b790-9ce5b9d9477f)
(I get one error for each step)

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-progressindicator--playground

### Steps to reproduce

When i scanned the carbon's playground sample of the ProgressIndicator component, i was able to see the error listed as well (same in my env)

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
